spec.bs

```diff
@@ -18,3 +18,91 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 Assume Explicit For: yes
 </pre>
+
+<h2 id="intro">Introduction</h2>
+
+<em>This section is non-normative</em>
+
+FLEDGE serves remarketing and custom audience use cases and is designed so that it cannot be used by third parties 
+to track user browsing behavior across sites.
+
+The FLEDGE API enables on-device auctions in the browser to choose relevant ads from websites the user has previously visited.
+
+
+<h2 id="join-ad-interest-group">joinAdInterestGroup</h2>

```

Comment:
Try to make section headers phrases rather than just the names of functions. So, this one might be

```suggestion
<h2 id="joining-interest-groups">Joining Interest Groups</h2>
```

Do this for the other headers too.